.github/workflows/create_baseline.yaml

```
@@ -0,0 +1,46 @@
+name: "CovidHub-baseline"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 19 * * 3"
+
+jobs:
+  generate-baseline-forecasts:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: r-lib/actions/setup-r@v2
+      with:
+        install-r: false
+        use-public-rspm: true
+      
+    - name: Install dependencies
+      run: |
+        install.packages(c("readr", "dplyr", "tidyr", "purrr", "checkmate", "cli", "lubridate", "remotes", "genlasso"))
+        remotes::install_github("cmu-delphi/epiprocess")
+        remotes::install_github("cmu-delphi/epipredict")
+      shell: Rscript {0}
+  
+    - name: generate baseline
+      run: |
+        Rscript src/code/get_baseline.r
+
+    - name: Commit changes
+      uses: EndBug/add-and-commit@v9
+      with: 
+        message: "Add baseline forecasts"
+        default_author: github_actions
+        push: true
```

I think we need this, no?
```suggestion
        push: true
        new_branch: auto-baseline
```

---

not sure but I will add it to be safe!